src/test/bench/testfilestore_backend.cc

```diff
@@ -57,9 +57,10 @@ void TestFileStoreBackend::write(
 
   os->queue_transaction(
     osr,
-    t,
+    std::move(*t),
```

Comment:
Yeah, thanks..I missed it, will fix..Basically, I searched for delete context that is been there in ObjectStore.h (I removed that)..It seems tests are creating their own delete context here.